packages/contracts/src/dollar/StakingShare.sol

```diff
@@ -1,17 +1,17 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.8.16;
 
-import "./ERC1155SetUri/ERC1155SetUri.sol";
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
-import "./ERC1155SetUri/ERC1155BurnableSetUri.sol";
-import "./ERC1155SetUri/ERC1155PausableSetUri.sol";
+import "@openzeppelin/contracts/token/ERC1155/extensions/ERC1155URIStorage.sol";
+import "@openzeppelin/contracts/token/ERC1155/extensions/ERC1155Burnable.sol";
+import "@openzeppelin/contracts/token/ERC1155/extensions/ERC1155Pausable.sol";
 import "./core/UbiquityDollarManager.sol";
 import "./utils/SafeAddArray.sol";
 
 contract StakingShare is
-    ERC1155SetUri,
-    ERC1155BurnableSetUri,
-    ERC1155PausableSetUri
+    ERC1155URIStorage,
```

Comment:
```suggestion
    ERC1155UriStorage,
```

I noticed this was updated in @zgorizzo69 pull request the other week but I think that we should have this be `Uri` instead of `URI` to match our code style after changing `NFT` to `Nft` in identifiers. 

I understand that this is not the fault of this PR.